src/components/ChatEntry.js

```diff
@@ -1,22 +1,42 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const senderClass =
+    props.sender === 'Estragon' ? 'chat-entry remote' : 'chat-entry local';
```

Comment:
NIce way to figure out whether to apply the local or remote css class. But what if the chat participants were other senders? Could we determine the "local sender" higher up and then pass that information down so that we can avoid hard-coding a name?